Reference/Events/Application-Startup.md

```diff
@@ -2,12 +2,36 @@
 versionFrom: 8.0.0
 ---
 
-# Application StartUp
-
-The ApplicationEventHandler approach for registering events has been removed in Umbraco V8, The new approach for registering custom code at 'ApplicationStarted' / 'ApplicationStarting' uses a combination of 'Components' and 'Composers', you can find examples on the [Composing](../../Implementation/Composing) page.
+# Application startup
+The `ApplicationEventHandler` approach for registering events has been removed in Umbraco 8. The new approach for registering custom code at the 'ApplicationStarting' and 'ApplicationStarted' events uses a combination of 'Composers' and 'Components', you can find a basic example below and more detailed ones on the [Composing](../../Implementation/Composing) page.
```

Comment:
Shouldn't 'ApplicationStarting' and 'ApplicationStarted' be formatted as code instead?
So, `ApplicationStarting` and `ApplicationStarted` ? 🤔

Reply:
It wasn't already, but we probably should indeed:

```suggestion
The `ApplicationEventHandler` approach for registering events has been removed in Umbraco 8. The new approach for registering custom code at the `ApplicationStarting` and `ApplicationStarted` events uses a combination of 'Composers' and 'Components', you can find a basic example below and more detailed ones on the [Composing](../../Implementation/Composing) page.
```